python/examples/integrations/flask_streaming/flask_with_whylogs.ipynb

```diff
@@ -0,0 +1,836 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# Integrating Whylogs into your Flask Flow\n",
+    "\n",
+    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/whylabs/whylogs/blob/mainline/python/examples/integrations/flask_with_whylogs.ipynb)\n",
+    "\n",
+    "Now that you've gone through the basics, let's get into how to integrate whylogs into your current work flow. Today, let's look at Flask to build a web app that wil get data from IRIS and us it for a prediction. Notice that we will log both the imput received and the prediction! "
+   ]
```

Comment:
We also don't have an overview section, where we actually explain what's going on. How whylogs and Flask interact with each other? When is data being logged, and where is the profile going to?
You could use it as reference the __overview__ section of this old blog post: https://towardsdatascience.com/deploy-and-monitor-your-ml-application-with-flask-and-whylabs-4cd1e757c94b
It could be a simpler explanation, but I think we should have it.